pkg/grafana/folder-handler.go

```diff
@@ -87,6 +88,58 @@ func (h *FolderHandler) GetUID(resource grizzly.Resource) (string, error) {
 	return resource.Name(), nil
 }
 
+type folderForSorting struct {
+	UID      string
+	Resource grizzly.Resource
+}
+
+// Sort sorts according to handler needs
+func (h *FolderHandler) Sort(resources grizzly.Resources) grizzly.Resources {
```

Comment:
Would be pretty easy to write a unit test for this. I can do it if you don't want to spend the time 😄

Reply:
It would be too easy to leave it to you. Not good for my character. I'll take a go!

Reply:
Done!